tests/DependencyInjection/SchebTwoFactorExtensionTest.php

```diff
@@ -107,6 +107,30 @@ public function load_fullConfig_setConfigValues(): void
         $this->assertHasParameter(['127.0.0.1'], 'scheb_two_factor.ip_whitelist');
     }
 
+    /**
+     * @test
+     */
+    public function load_envVarBasedFullConfig_setConfigValues(): void
+    {
+        $config = $this->getEnvVarBasedFullConfig(true);
+        $this->extension->load([$config], $this->container);
+
+        $this->assertHasParameter(true, 'scheb_two_factor.trusted_device.enabled');
+        $this->assertHasParameter(true, 'scheb_two_factor.backup_codes.enabled');
+        $this->assertHasParameter(true, 'scheb_two_factor.google.enabled');
+        $this->assertHasParameter(true, 'scheb_two_factor.totp.enabled');
+        $this->assertHasParameter(true, 'scheb_two_factor.email.enabled');
```

Comment:
Except for `scheb_two_factor.trusted_device.enabled`, these parameters don't exist. Instead, check for service definitions which should be set when the feature is enabled. For example, when email authentication is enabled, a service definition `scheb_two_factor.security.email.provider` should be defined. If it's disabled, this one isn't defined. For each of the feature toggles there is such services that you can use to check if the feature has been activated or not. Check the test methods for examples. One service defintion per feature flag is enough, no need to test all of them for each feature flag.